ui/public/style.css

```diff
@@ -33,6 +33,15 @@ body {
     100% { transform: rotate(360deg); }
 }
 
+/* Records list table */
+
+td {
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  max-width: 65px;
+}
+
```

Comment:
That's actually what this commit fixes :) Limiting the table width has no effect.